eZ/Publish/Core/Repository/ContentService.php

```diff
@@ -522,7 +522,7 @@ public function createContent(APIContentCreateStruct $contentCreateStruct, array
         }
 
         if ($contentCreateStruct->alwaysAvailable === null) {
-            $contentCreateStruct->alwaysAvailable = false;
+            $contentCreateStruct->alwaysAvailable = $contentCreateStruct->contentType->defaultAlwaysAvailable ?? false;
```

Comment:
`??` is PHP 7 only

Reply:
It's easy to get use to nice things ;)

Reply:
Well, but for older versions we still could keep it shorter:
```php
 $contentCreateStruct->contentType->defaultAlwaysAvailable ?: false;
```
:stuck_out_tongue: